packages/card/resource-card/index.tsx

```diff
@@ -25,6 +25,12 @@ export function ResourceCard({
   thumbnail,
   appearance,
 }: ResourceCardProps): JSX.Element {
+  const meta: string[] = [category]
+
+  if (date) {
+    meta.unshift(date)
```

Comment:
Instead of mutating an array, let's use something like this:

```ts
const meta: string[] = date ? [date, category] : [category]
```

Reply:
Ah, another good immutability catch 🚀 Addressed here:
https://github.com/hashicorp/react-components/pull/1001/commits/dd8b04a63f301a9cdb2a78a4f0c001e0ae784eeb